fea: create project readme (#68)
podcodar/webapp: README.md

> # Hello Visitor !! Welcome in PodCodar web App

Reviewer: I believe a more correct form for this sentence would be  "Welcome to the PodCodar web App"

> It's nice have you for here! This is PodCodar web a project of the community PodCodar with it's main objective is to teach good practices of development for our mentees.

Reviewer: It's nice have you for here! -> It's nice have you here!

This is PodCodar web a project of the community PodCodar with it's main objective is -> This is  a project created by the PodCodar community and it's main objective is....

### Technologies

<p align="left">

> <a href="https://www.typescriptlang.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/typescript/typescript-original.svg" alt="typescript" width="40" height="40" paddingLeft="5px"/>  </a>

Reviewer: Usually, keeping links like this can be boring, because, if the original source stop serving this file it will appear broke in our documentation, demanding new sources and changes in the doc.

As a suggestion, I believe you can download the images you're using, and we can create a folder like `docs/images` where we can upload those images and free ourselves from this condition^^

Reviewer 2: Great suggestions.

Another thing to avoid is HTML tags. We should always preferer Markdown notation over HTML tags

  <img src="./public/images/react.svg"height="30" />
  <img src="./public/images/next.svg"height="30" />
  <img src="./public/images/node.svg"height="30" />
  <img src="./public/images/git.svg"height="30" />
  <img src="./public/images/linux.svg"height="30" />
</div>

### Partners

[![image](./public/images/vercel-logo.svg)][1]

[1]: https://vercel.com/?utm_source=podcodar&utm_campaign=oss

### Start Project

- Install dependencies using library yarn whit command `yarn`.

> - Create archive `.env.local` in root folder and fill in the following fields :

Reviewer 2: Suggested text: "Create a file..."
```suggestion
- Create a file `.env.local` in root folder and fill in the following fields :
```

>         - FIREBASE_APP_ID="***"

Reviewer 2: Where is the code markation to hold the file content? Always use ``` for code blocks.

        - FIREBASE_API_KEY="***"
        - FIREBASE_PROJECT_ID="***"
        - FIREBASE_MESSAGING_ID="***"
        - FIREBASE_MEASUREMENT_ID="***"

>   after request credentials for team member.

Reviewer 2: ```suggestion
- After this, ask a team member for the real credentials and replace the values on `.local.env` file.
```

> ### Start Project

Reviewer: I beliave that here whe sould keep the actual usage of our project, for example:

```bash
yarn
yarn dev
```

> - now enter a command `next start` and your project will be running.

Reviewer: yarn instead of this line^^

Reviewer 2: Let's avoid using the `next` command directly. If we use it directly, we need to install it as a global package instead of a local one.

# Hello Visitor !! Welcome to the PodCodar WebApp

> It's nice have you here! This is a project created by the PodCodar community and it's main objective is to teach good practices of development for our mentees.

Reviewer 2: ```suggestion
It's nice to have you here! This project is the web app created by the PodCodar community, and its main objective is to teach good practices of development for our mentees.
```

### Technologies

<div  style=" max-width:200px ;display:grid; grid-template-columns:repeat(6,auto);grid-gap:1rem">
  <img src="./packages/docs/img/typescript.svg"height="30" />
  <img src="./packages/docs/img/react.svg"height="30" />
  <img src="./packages/docs/img/next.svg"height="30" />
  <img src="./packages/docs/img/node.svg"height="30" />
  <img src="./packages/docs/img/git.svg"height="30" />
  <img src="./packages/docs/img/linux.svg"height="30" />
</div>

### Partners

[![image](./packages/docs/img/vercel-logo.svg)][1]

### Start Project

- Install dependencies using library yarn whit command `yarn`.

> - Create a file `.env.local` in root folder and fill in the following fields

Reviewer 2: ```suggestion
- Create a file `.env.local` in the root folder and fill in the following fields:
```

Reviewer 2: And I still think we should add the fields as a file template:

```shell
# file: .env.local

FIREBASE_APP_ID="***"
FIREBASE_API_KEY="***"
FIREBASE_PROJECT_ID="***"
FIREBASE_MESSAGING_ID="***"
FIREBASE_MEASUREMENT_ID="***"
```

  - FIREBASE_APP_ID=`***`

  - FIREBASE_API_KEY=`***`

  - FIREBASE_PROJECT_ID=`***`

  - FIREBASE_MESSAGING_ID=`***`

  - FIREBASE_MEASUREMENT_ID=`***`

>   After this, ask a team member for the real credentials.

Reviewer 2: ```suggestion
- After creating it, ask a team member for the real credentials.
```

> - now enter a command `yarn start` and your project will be running.

Reviewer 2: ```suggestion
- now enter a command `yarn dev` and your project will be running.
- to run de **production** build `yarn build && yarn start`
```

### Our Wikis

> [Web app wiki][2]
> 
> [Engineering Wiki][3]
> 
> [Project Board][4]

Reviewer 2: ```suggestion
- [Web app wiki][2]
- [Engineering Wiki][3]
- [Project Board][4]
```

### Let's code with us

[1]: https://vercel.com/?utm_source=podcodar&utm_campaign=oss

> [2]: https://www.notion.so/podcodar/PodCodar-WebApp-4c1b6a896ebf45b49fed62c28ac45bef

Reviewer 2: noice ❤️

FIREBASE_MEASUREMENT_ID="***"
```

> After creating it, ask a team member for the real credentials.

Reviewer 2: ```suggestion
- After creating it, ask a team member for the real credentials.
```

- now enter a command `yarn dev` your project will be running.

- to run de **production** build `yarn build && yarn start`

### Our Wikis

[Web app wiki][2]

[Engineering Wiki][3]

> [Project Board][4]

Reviewer 2: This should be a list, not isolated links
